lib/cache/access_list_test.go

```diff
@@ -175,3 +178,76 @@ func TestAccessListReviews(t *testing.T) {
 		deleteAll: p.accessLists.DeleteAllAccessListReviews,
 	})
 }
+
+func TestCountAccessListMembersScoping(t *testing.T) {
+	modules.SetTestModules(t, &modules.TestModules{
+		TestBuildType: modules.BuildEnterprise,
+		TestFeatures: modules.Features{
+			Entitlements: map[entitlements.EntitlementKind]modules.EntitlementInfo{
+				entitlements.Identity: {Enabled: true},
+			},
+		},
+	})
+
+	p := newTestPack(t, ForAuth)
```

Comment:
`newTestPack` currently just calls `newPack` and does a `require.NoError` but doesn't have a way to forward a list of `packOption`s such as `memoryBackend(true)` which would speed this up a bit compared to using a sqlite backend.

Adding `opts ...packOption` and `opts...` in the appropriate places would let you add `memoryBackend(true)` here.

```suggestion
	p := newTestPack(t, ForAuth, memoryBackend(true))
```